### DISCUSSION TITLE
Refresh not working.

### DISCUSSION THREAD

**Discussion by Julio-N-Coder** [vercel/next.js#47536] [Help]
### Summary

I am tying to use next.js for the first time so I followed there create your first app on the documentation. Things were going well until the point where I had to edit the page. When I edited the page, the page was not refreshing to what I wrote even after I saved it. I tried so many things, and I can't figure out why it is not working. I hope someone can help me because I am tired of this and I just want to learn. One thing I will say is that it only changes when I stop the server and run it again.

**Comment by lajuma** [comment]
I have the same problem, just followed the tutorial. Can you please tell how you resolved the issue when marking as closed?!
